addons/web/static/src/js/public/public_notification.js

```diff
@@ -0,0 +1,10 @@
+odoo.define('web.public.Notification', function (require) {
+"use strict";
```

Comment:
Here `'` instead of `"`, this is a new file so you can follow our conventions and our convention is `'` for "technical strings" and `"` for strings which are show to the user (not my invention this one ^^).